changelog/46.fixed.md

```diff
@@ -0,0 +1 @@
+Added try/except logic for Azure Authentication methods.
```

Comment:
**issue (blocking):** this is one of the things that you _did_... what issue did you _fix_?

**Example:** Fixed TypeError for ManagedIdentityCredential when using service principal credentials.